docker/Dockerfile

```diff
@@ -199,15 +200,13 @@ RUN echo "Building Avalon SDK\n" \
 # Build Final image and install dependent modules
 FROM base_image as final_image
 
-RUN apt-get install -y -q \
-    git
-
-RUN apt-get install -y -q python3-pip \
+RUN apt-get update \
+ && apt-get install -y -q python3-pip git \
  && pip3 install json-rpc web3 py-solc-x nose2 \
  && echo "Install fabric python sdk\n" \
  && pip3 install virtualenv \
  && pip3 install git+https://github.com/hyperledger/fabric-sdk-py.git \
- && apt-get autoremove --purge -y -q python3-pip
+ && apt-get autoremove --purge -y -q python3-pip git
```

Comment:
Why is git being added ? Any other issue as well?

Reply:
No other issue. If you see, git is being installed in line 203. I deleted those lines and adding it again. So that it can be removed along with other unwanted package.